.github/workflows/ctag_DY_workflow.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.9","3.10"]
```

Comment:
Hi @Ming-Yan you might have missed the discussion [here](https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/50#discussion_r1135770669), please change to keeping 3.8 and 3.10

Reply:
ah right, I missed the discussion. How about the environment here? also keep 3.8,3.9.3.10?

https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/50#discussion_r1135767829

Reply:
yes :+1:  and then the PR should be ready for merge